Instagram/app/controllers/images_controller.rb

```diff
@@ -0,0 +1,57 @@
+class ImagesController < ApplicationController
+
+  def index
+    # @images = Image.all
+    @images = Image.all.paginate(:page => params[:page], :per_page => 9)
+  end
+
+  def new
+    @image = Image.new
+  end
+
+  def create
+    @image = Image.new(image_params)
+    @image.save
+    @images = Image.paginate(page: params[:page])
+      last_page = (@images.length.to_f/9).ceil
```

Comment:
Indentation is off here, likewise — break this habit.

Keep using rubocop until you reliably get clean runs consistently. This is a good config to use https://github.com/makersacademy/scaffolint